website/blog/2023-12-04-0.73-debugging-improvements-stable-symlinks.md

```diff
@@ -0,0 +1,178 @@
+
+Symlinks are deeply represented in Metro's internals, meaning they work with features such as [Fast Refresh](/docs/fast-refresh), and incur little performance overhead during bundling. Symlinks are supported on all desktop platforms, with and without Watchman.
+
+:::info
+#### Monorepo workarounds
+
+We are aware there are still edge cases when using React Native in a monorepo layout. We have planned work to address some of these, which didn't quite make it into 0.73 — but which we aim to ship as soon as possible.
+
+**For React Native template projects (`npx react-native init`)**, you will need to configure any [`watchFolders`](https://metrobundler.dev/docs/configuration/#watchfolders) outside of the project root in order for Metro to discover them ([more info](https://metrobundler.dev/docs/configuration/#unstable_enablesymlinks-experimental)). You may also need to update file paths if your `react-native` dependency is installed to a folder at a different level.
+
+**For Expo apps**, support for Yarn (Classic) workspaces is configured out of the box. See also the [Work with monorepos](https://docs.expo.dev/guides/monorepos/) guide in the Expo docs.
+:::
+
+### Kotlin Template on Android
+
+We're excited to announce that, starting from 0.73, Kotlin is now the **recommended language** for Android apps built with React Native. This follows the direction that the Android ecosystem has been moving in for several years and allows you to write your app using a modern language.
+
+We've updated React Native's template on Android to use Kotlin instead of Java. The new `MainActivity.kt` and `MainApplication.kt` files are 36% smaller in size.
+
+The upgrade helper has also been updated to make it easier to migrate your `.java` file to `.kt` files.
+
+### Android 14 Support
+
+We've updated some capabilities to support Android 14.
+
+#### Grant partial access to photos and videos
+
+[Selected Photos Access](https://developer.android.com/about/versions/14/changes/partial-photo-video-access) allows Android 14 users to grant apps access to specific items in their media library, rather than access to all media. In 0.73, React Native apps now support this capability, by using the `READ_MEDIA_VISUAL_USER_SELECTED` permission in the [`PermissionsAndroid`](https://reactnative.dev/docs/permissionsandroid) API.
+
+#### Context-registered broadcasts
+
+When using SDK >= 34, we've migrated [Broadcast Receivers](https://developer.android.com/guide/components/activities/process-lifecycle) to support context-registered broadcasts — part of the app lifecycle model — in Android 14. This means React Native apps will behave predictably on Android 14 devices.
+
+#### Target SDK version
+
+In 0.73, React Native developers can now target the latest Android SDK version, [API Level 34](https://developer.android.com/guide/topics/manifest/uses-sdk-element?hl=en#ApiLevels) (*Upside Down Cake*).
+
```

Comment:
```suggestion
#### Min SDK Bump

React Native 0.73 will be the latest version to support Android 5.0 (API Level 21). The upcoming version of React Native will have a min SDK of 23 (Android 6.0). You can read more about the upcoming changes to min SDK bump [over here](TODO). 
```

cc @huntie we need to add a link here

Reply:
@cortinico Do you know what this should link to or is likely to be?